shared/chat/conversation/info-panel/menu/index.tsx

```diff
@@ -121,20 +122,29 @@ class InfoPanelMenu extends React.Component<Props> {
   render() {
     const props = this.props
     const isGeneralChannel = !!(props.channelname && props.channelname === 'general')
-    const addPeopleItems: Kb.MenuItems = [
-      {
-        icon: 'iconfont-mention',
-        onClick: props.onAddPeople,
-        style: {borderTopWidth: 0},
-        subTitle: 'Keybase, Twitter, etc.',
-        title: 'Add someone by username',
-      },
-      {
-        icon: 'iconfont-contact-book',
-        onClick: props.onInvite,
-        title: Styles.isMobile ? 'Add someone from address book' : 'Add someone by email',
-      },
-    ]
+    const hasChannelSection = !props.isSmallTeam && !props.hasHeader
+    const addPeopleItems: Kb.MenuItems = flags.teamsRedesign
+      ? [
+          {
+            icon: 'iconfont-new',
+            onClick: props.onAddPeople,
+            title: hasChannelSection ? 'Add/Invite people to team' : 'Add/invite people',
+          },
+        ]
+      : [
+          {
+            icon: 'iconfont-mention',
+            onClick: props.onAddPeople,
+            style: {borderTopWidth: 0},
+            subTitle: 'Keybase, Twitter, etc.',
+            title: 'Add/Invite',
```

Comment:
why no "people" here vs above?

Reply:
Oh I meant to leave this as "Add someone by username" actually, wanted not to change things with the flag off